<launch>
    <node name="depthai_sonar_spatial_detection" pkg="cv" type="depthai_sonar_spatial_detection.py" output="screen"/>
</launch>
